src/main/java/com/google/sps/data/Attendee.java

```diff
@@ -0,0 +1,56 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import java.util.Optional;
+import com.google.appengine.api.datastore.Entity;
+
+/** Class that represents an attendee. */
+public class Attendee {
+
+  private String sessionId;
+  private String screenName;
+  private Date timeLastPolled;
```

Comment:
I am thinking that the first two could be final, but I am not sure with the timeLastPolled, since it's value is changing as client polls.